docs/syntax.rst

```diff
@@ -2,9 +2,14 @@
 The Guide To The FTL Syntax
 ===========================
 
-FTL is an open source, localization-specific scripting language used to process 
-gender, plurals, conjugations, and most of the other quirky elements of natural 
-language.
+FTL is a localization language used for describing translation resources.
+
+It assigns messages in a given language to unique identifiers used by
+the software to retrieve the message.
+
+FTL is designed to be simple to read, but allow to represent complex concepts
+from natural languages like gender, plurals, conjugations,
+and most of the other quirky elements.
```

Comment:
Drop the last part (`and most..`) and:

    from natural languages like genders, plurals, grammatical cases, conjugations and others.